docs/spfx/web-parts/isolated-web-parts-retirement.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Isolated web parts retirement
+description: All datails on the domain isolated web parts retirement
+ms.date: 04/23/2024
+ms.localizationpriority: high
+---
+
+# Retiring SharePoint Framework domain isolated web parts for SharePoint Online
+
+After careful consideration and extensive review of usage data and feedback we’ve decided to announce the retirement of the domain isolated web parts feature in SharePoint Framework (SPFx). As of April 2nd, 2026 domain isolated web parts will be fully retired and stop working for existing tenants. If tenants are still using domain isolated web parts then they’ll see the web part render and error message and stop functioning. New tenants onboarding from April 2nd, 2025 will not be able to use domain isolated web parts. This applies to all environments including Government Clouds and Department of Defense. There will not be an option to extend domain isolated web parts beyond April 2, 2026.
+
+We apologize for any inconvenience this may cause, but we believe this is the best decision for the future of SPFx and SharePoint.
+
+## Timeline
+
+Date | Action
+-----|-------
+April 2, 2025 | Domain isolated web parts will be turned off for newly created tenants.
+April 2, 2026 | Domain isolated web parts will be turned off for existing tenants.
+
+## Overview
+
+Domain isolated web parts allow developers to create web parts that run in a separate domain from the host page, providing an additional layer of security and isolation. Due to the performance implications of running in dedicated domain we saw limited usage of this feature.
+
+To understand whether you have installed domain isolated web parts and granted permissions you can verify if the [SharePoint admin center API access page](https://learn.microsoft.com/sharepoint/api-access) contains permissions listed under the “Isolated” grouping.
+
+![SharePoint admin center showing the permissions granted for domain isolated web parts](../../images/discoverdomainisolatedwebpartusage.png)
+
+If you see permissions listed under **Isolated** then use the instructions below to understand more about the deployed domain isolated web parts. This script will create two CSV files:
+
+- **Solutions.csv**: this file contains an overview of the solutions that contain domain isolated web parts
+- **Pages.csv**: this file contains an entry per page holding one of the domain isolated web parts
+
+To use this script follow these steps:
+
+1. Copy PowerShell script file contents (end of this page) and save it as `Find-DomainIsolatedWebparts.ps1` on your local drive.
+2. [Install PnP PowerShell](https://pnp.github.io/powershell/articles/installation.html) in case that is not yet available.
```

Comment:
Numbered bullets should always be `1.` and not explicitly numbered. Allow for the rendering engine to number them so numbers don't get out of sync.